specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/standardAssignments.json

```diff
@@ -0,0 +1,440 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - standard assignments was deleted"
+          },
+          "204": {
+            "description": "No Content - standard assignments does not exist"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/standardAssignments": {
+      "get": {
+        "x-ms-examples": {
+          "List standard assignments": {
+            "$ref": "./examples/StandardAssignments/ListStandardAssignments.json"
+          }
+        },
+        "tags": [
+          "StandardAssignments"
+        ],
+        "description": "Get a list of all relevant standard assignments over a scope",
+        "operationId": "StandardAssignments_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StandardAssignmentsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StandardAssignmentsList": {
+      "type": "object",
+      "description": "Page of a standard assignment list",
+      "properties": {
+        "value": {
+          "description": "Collection of standardAssignments in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StandardAssignment"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "StandardAssignment": {
+      "type": "object",
+      "description": "Security Assignment on a resource group over a given scope",
+      "properties": {
+        "properties": {
+          "description": "Properties of a standard assignments assignment",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StandardAssignmentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
```

Comment:
>v1 [](http://example.com/codeflow?start=35&length=2)

why not v5 ?

Reply:
Fixed.

Reply:
We are using our definition for resource (as you can see in the path)
this is the standard for all our RP is security. there isn't v5 for it.